nomenclature/testing.py

```diff
@@ -40,12 +40,12 @@ def assert_valid_structure(
     ----------
     path : Path
         directory path to the file of interest
-    dimensions : List[str]
-        Optional list of dimensions to be checked
-    mappings : str
-        Optional non-default name for the mappings folder, default "mappings"
-    definitions : str
-        Optional non-default name for the definitions folder
+    dimensions : List[str], optional
+        List of dimensions to be checked
+    mappings : str, optional
+        Name for the mappings folder
```

Comment:
The name of the standard folder is not clear from the function signature (because it says `None`), so better to repeat it here for clarity.
```suggestion
        Name for the mappings folder, defaults to "mappings"
```